libmscore/CMakeLists.txt

```diff
@@ -71,6 +71,8 @@ add_library (
       read206.cpp
       read300.cpp stafftypelist.cpp stafftypechange.cpp
       bracketItem.cpp
+      standardfactory.cpp jianpufactory.cpp jianpuchord.cpp jianpunote.cpp jianpurest.cpp 
+      jianpuhook.cpp jianpubeam.cpp jianpunotedot.cpp
```

Comment:
Thanks, Jojo.  It was my dumb mistake to have forgotten to add new files.  It is fixed now in the new push. Please take a look.